stable/jenkins/CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version 1.5.7 is auto generated based on git commits. Those include a reference to the git commit to be able to get more details.
 
+## 1.25.0 Update plugin versions & sidecar container
+
+| plugin                | old version | new version |
+| --------------------- | ----------- | ----------- |
+| kubernetes            | 1.25.3      | 1.25.7      |
+| workflow-job          | 2.38        | 2.39        |
+| credentials-binding   | 1.21        | 1.22        |
+| configuration-as-code | 1.39        | 1.41        |
+
+configAutoReload container updated from `kiwigrid/k8s-sidecar:0.1.132` to `kiwigrid/k8s-sidecar:0.1.144`
```

Comment:
looks like you missed the `k8s-sidecar` update in `values.yaml` :-)

Reply:
thanks for catching this. Forgot to add it ;-)

Reply:
there's a merge conflict line in the CHANGELOG which can be removed later
`>>>>>>> master`